shared/desktop/app/node.desktop.tsx

```diff
@@ -73,8 +73,11 @@ const focusSelfOnAnotherInstanceLaunching = (_, commandLine) => {
 
   // The new instance might be due to a URL schema handler launch.
   logger.info('Launched with URL', commandLine)
-  if (commandLine.length > 1 && commandLine[1] && commandLine[1].startsWith('web+stellar:')) {
-    sendToMainWindow('dispatchAction', {payload: {link: commandLine[1]}, type: ConfigGen.link})
+  if (commandLine.length > 1 && commandLine[1]) {
+    const link = commandLine[1]
+    if (link.startsWith('web+stellar:') || link.startsWith('keybase://')) {
+      sendToMainWindow('dispatchAction', {payload: {link}, type: DeeplinksGen.link})
+    }
```

Comment:
If I understand correctly, this is only checking argv1, i.e. the calling mechanism must be e.g. `./Keybase keybase://chat/username`. (And I guess this was true for the web+stellar:// links before this PR.)

This means you cannot use electron's advertised alternative invocation method: `/usr/lib/electron/electron path-to-app keybase://chat/username` (because it is in argv2 in this case).
As such, the Arch Linux package I maintain which uses a generic electron binary does not register these command-line options.

Is there something that can be done about this?